test/silhouette.jl

```diff
@@ -34,4 +34,21 @@ c = [3, 1]
 
 @test silhouettes(a, c, D) ≈ [0.5, 0.5, -1/3, 0.0]
 
+@testset "silhouette() errors when degenerated clustering is given" begin
+    a = [fill(1, 5); fill(2, 5)]
+    d = fill(0, (10, 10))
+
+    @test silhouettes(a, d) ≈ fill(0.0, 10)
```

Comment:
Sorry, overlooked that at the first iteration: I think here we want an exact (`==`) comparison.

Reply:
While knowing internal implementation of Julia type conversations of Julia it's true that `==` holds it still may not be advisable to compare floating point quantities with equality unless your scope of testing is for ensuring Julia division operator is behaving `x/y` properly for type conversions. Request you to reconsider this change suggested.